application/modules/settings/views/partial_settings_updates.php

```diff
@@ -29,10 +29,33 @@
                 }, checktime);
             },
             error: function(data) {
-                <?php echo(IP_DEBUG ? 'console.log(data);' : ''); ?>
-                $('#updatecheck-loading').addClass('hidden');
-                $('#updatecheck-failed').removeClass('hidden');
-            },
+                $.ajax({
```

Comment:
Maybe fix the indentation before merge

Reply:
Fixed by https://github.com/InvoicePlane/InvoicePlane/pull/699/commits/c43f25690845c219b4ebfb1fac6be06db61ea53b